app/models/instance_user.rb

```diff
@@ -2,6 +2,8 @@
 class InstanceUser < ApplicationRecord
   include InstanceUserSearchConcern
   acts_as_tenant :instance, inverse_of: :instance_users
+  after_initialize :set_defaults, if: :new_record?
+  before_validation :set_defaults, if: :new_record?
```

Comment:
I think it is not necessary to `set_defaults` twice, calling it at `after_initialize` should be sufficient.